### PR TITLE
badge-component

### PR DESCRIPTION
**Pull request by AmishMishra11**
complete badge component

### CODE REVIEW

**Review by abhijeetekad — APPROVED** [review]
Looks great. You can reduce header height later. Now you are ready to merge.

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  text-align: center;
+  color: white;
+  background-color: var(--primary-color);
+}
+.icon-bell {
+  position: relative;
+}
+.icon-bell p {
+  position: absolute;
+  top: -0.9rem;
```
Comment (Prabhav-Sharma) [inline components/badge/badge.css]: Not good practise to use rems below 1, convert to absolute units

**Review by Prabhav-Sharma — CHANGES_REQUESTED** [review]
Can increase the green badge size on avatars. Rest all is good

File: components/badge/badge.css (hunk 1 above)
Comment (AmishMishra11) [reply components/badge/badge.css]: yes thank you i've fixed it can you review it again

**Pull request merged**